.mergify.yml

```diff
@@ -1,8 +1,10 @@
 pull_request_rules:
   - name: Automatic merge on approval
     conditions:
-      - "status-success=checks/approved"
-      - "#approved-reviews-by>=2"
+      - "#approved-reviews-by>=1"
```

Comment:
I would suggest not changing this option and leaving it to 2 or more because I think having more reviews on a PR is always great because this gets more people's eyes on the changes that were made. Which can help get more ideas on the PR as wells as sometimes make the PR much better. :slightly_smiling_face: